docs/docs/artifact-store.md

```diff
@@ -150,25 +143,29 @@ Server: Jetty(8.1.11.v20130520)
 ```
 
 
-## Automatic Artifact Storing
+## Automatic Artifact Storage
+
+An AppDefinition holds a sequence of URIs that Marathon fetches when it starts each app instance.
 
-An AppDefinition holds a sequence of URIs, that get fetched on each instance, that gets started.
 The artifact could be fetched directly from the source, or put into the artifact store.
 One simple way to do this is automatic artifact storing.
 
-The AppDefinition has a field storeUrls, which holds an array of URL strings.
+You can automatically store artifacts in the artifact store with the ```storeURLs``` field of the AppDefinition. The ```storeURLs``` field holds an array of URL strings.
 Every URL here is processed in this way:
 
-* The URL gets downloaded
-* The byte stream is stored in the asset store
-* The asset store url is added to the AppDefinition uris list
-* The url is removed from the storeUrls array
+* The URL is downloaded. 
+* The byte stream is stored in the asset store.
+* Marathon adds the asset store URL to the AppDefinition URIs list.
+* Marathon removes the URL from the storeUrls array.
 
-As a result, all storeUrls are accessible from the artifact store.
-All instances that will run the application, will load the needed assets from the artifact store.
+As a result, all storeUrls are accessible from the artifact store. All instances that run the application are able to load the necessary assets from the artifact store.
 
+### Prerequisites
+
+To use this feature, all assets need to be resolvable by marathon itself.
+The HTTP server should support HTTP ETag Header in order to circumvent manual content hash creation.
 
-### Create an application definition with automatic artifact resolution
+### Create an Application Definition with Automatic Artifact Resolution
```

Comment:
Automatic here means: do not use the /v2/artifacts API, but the storeURLs in the app definition (which handles all the details)